### DISCUSSION TITLE
GPU performance on the server

### DISCUSSION THREAD

**Discussion by dodedov** [ltsp/ltsp#571] [Q&A] [answered]
Hello! How important is the GPU performance on the server for the comfortable work of clients?

If we assume that the server renders the program interface instead of the client, is it true that a productive GPU will facilitate the work of the CPU and make the work more comfortable for customers?

**Comment by alkisg** [comment, marked as answer]
You're thinking "thin clients" while the new LTSP is about "netbooted diskless workstations" which run the programs in their own CPU/GPU.

You can read more about thin clients in https://github.com/ltsp/ltsp/discussions/243.

But in any case, the server GPU doesn't matter, it doesn't even need to have one.

**Reply by dodedov** [reply]
Thanks, I got it! I confused the X2Go function with LTSP. I thought LTSP could do the same thing.

**Comment by alkisg** [comment]
LTSP is used to netboot diskless workstations.
After they're booted, you can run x2go or xfreerdp or any other software you want on them.

**Reply by dodedov** [reply]
Yes right. So I want to try. Thanks for the help.
